src/app/setup-wizard/pre-visit/pre-assessment-setup/pre-assessment-setup.component.html

```diff
@@ -128,32 +128,20 @@ <h6>Utility Types</h6>
-                                                            <div class="input-group">
-                                                                <input name="{{'energyUse_'+assessment.guid}}"
-                                                                    type="number" class="form-control"
-                                                                    [disabled]="true"
-                                                                    id="{{'energyUse_'+assessment.guid}}"
-                                                                    [value]="assessment.energyUse | number:'1.0-2'">
-                                                                <span class="input-group-text"
-                                                                    [innerHTML]="companyEnergyUnit | unitsDisplay">
-                                                                </span>
-                                                            </div>
+                                                        <div class="col-sm-7" id="{{'energyUse_'+assessment.guid}}">
+                                                            {{assessment.energyUse | number:'1.0-2'}}&nbsp;
+                                                            <span
+                                                                [innerHTML]="companyEnergyUnit | unitsDisplay">
+                                                            </span>
                                                         </div>
                                                     </div>
 
                                                     <div class="row">
                                                         <label class="col-sm-5 col-form-label"
                                                             for="{{'cost_'+assessment.guid}}">Annual
                                                             Costs</label>
-                                                        <div class="col-sm-7">
-                                                            <div class="input-group">
-                                                                <input name="{{'cost_'+assessment.guid}}" type="number"
-                                                                    class="form-control" [(ngModel)]="assessment.cost"
-                                                                    (input)="saveChanges(assessment)"
-                                                                    id="{{'cost_'+assessment.guid}}">
-                                                                <span class="input-group-text">&dollar;</span>
-                                                            </div>
+                                                        <div class="col-sm-7" id="{{'cost_'+assessment.guid}}">
+                                                            {{assessment.cost | number:'1.0-2'}}&nbsp;&dollar;
```

Comment:
for costs use the "currency" pipe that is built into angular. That way when we add localization it will handle that for us.

Reply:
Thanks, changed